package.json

```diff
@@ -52,22 +44,34 @@
     "@typescript-eslint/eslint-plugin": "^5.42.1",
     "@typescript-eslint/parser": "^5.42.1",
     "eslint": "^8.40.0",
-    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-jest": "^27.1.5",
     "eslint-plugin-jsdoc": "^39.6.2",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "jest": "^26.4.2",
+    "eslint-plugin-prettier": "^5.2.1",
+    "jest": "^29.7.0",
     "outdent": "^0.8.0",
+    "prettier": "^3.3.3",
+    "prettier-plugin-packagejson": "^2.5.2",
     "rimraf": "^3.0.2",
-    "ts-jest": "^26.5.6",
+    "ts-jest": "^29.2.5",
     "typescript": "~4.8.4"
   },
+  "peerDependencies": {
+    "prettier": "^3.3.3"
+  },
+  "packageManager": "yarn@3.2.4",
+  "engines": {
+    "node": ">=14.0.0"
+  },
```

Comment:
Supported versions were `^18.18 || >=20`, any reason we need v14?

Reply:
I messed this up after rebasing. Fixed it now.